css/style.css

```diff
@@ -0,0 +1,9 @@
+body {
+    max-width: 80%;
+    margin-left: 10%;
+    margin-right: 10%;
+  }
+form{
+    text-align: center;
+    background-color: beige;
+}
```

Comment:
- [ ] [OPTIONAL] Nice job being polite, and giving reasons for the requested change :+1:, however I suggest you also phrase your comments as suggestions instead of orders

Example:
:x: Wrong
Please, I think you should add a new line at the end of the CSS code because it makes your code clean.


:heavy_check_mark: Right
Please, I kindly suggest that you should add a new line at the end of the CSS code because it makes your code clean.